dev/com.ibm.websphere.appserver.thirdparty.eclipselink.2.7/resources/org/eclipse/persistence/exceptions/i18n/JSONExceptionResource.nlsprops

```diff
@@ -27,5 +27,5 @@
 #   (single quote must be coded as one single quote '). 
 #
 # -------------------------------------------------------------------------------------------------
-60001=Input JSON document is invalid or doesn't match target object graph.
+60001=Input JSON document is invalid or doesn''t match target object graph.
```

Comment:
Can we change this to be "does not" instead?

Reply:
@dazey3 I completely lost track of this, sorry, I've updated with your suggestion, can you take another PR review look please.